staging-next 2025-04-03

mweinelt opened the pull request:
##### Helpful links

- workflow docs: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#staging
- important jobs:
  https://hydra.nixos.org/job/nixpkgs/staging-next/unstable#tabs-constituents
  https://hydra.nixos.org/job/nixos/staging-next-small/tested
- jobsets:
  https://hydra.nixos.org/jobset/nixpkgs/staging-next
  https://hydra.nixos.org/jobset/nixos/staging-next-small
- nix-review reports: https://malob.github.io/nix-review-tools-reports/
- previous staging-next: https://github.com/NixOS/nixpkgs/pull/389579
- next staging-next: #401526

##### Important breakages
_nothing notable so far_

fabaff committed:
python313Packages.gios: 5.0.0 -> 6.0.0

Diff: https://github.com/bieniu/gios/compare/refs/tags/5.0.0...6.0.0

Changelog: https://github.com/bieniu/gios/releases/tag/6.0.0

fabaff committed:
python313Packages.nettigo-air-monitor: 4.0.0 -> 4.1.0

Diff: https://github.com/bieniu/nettigo-air-monitor/compare/refs/tags/4.0.0...4.1.0

Changelog: https://github.com/bieniu/nettigo-air-monitor/releases/tag/4.1.0

fabaff committed:
python313Packages.bleak-esphome: 2.11.0 -> 2.12.0

Diff: https://github.com/bluetooth-devices/bleak-esphome/compare/refs/tags/v2.11.0...v2.12.0

Changelog: https://github.com/bluetooth-devices/bleak-esphome/blob/v2.12.0/CHANGELOG.md

r-ryantm committed:
python312Packages.python-roborock: 2.12.2 -> 2.16.1

fabaff committed:
python313Packages.aiocomelit: 0.11.1 -> 0.11.3

Diff: https://github.com/chemelli74/aiocomelit/compare/refs/tags/v0.11.1...v0.11.3

Changelog: https://github.com/chemelli74/aiocomelit/blob/v0.11.3/CHANGELOG.md

fabaff committed:
python313Packages.meteofrance-api: 1.3.0 -> 1.4.0

Diff: https://github.com/hacf-fr/meteofrance-api/compare/refs/tags/v1.3.0...v1.4.0

Changelog: https://github.com/hacf-fr/meteofrance-api/releases/tag/v1.4.0

dotlambda committed:
home-assistant: 2025.3.4 -> 2025.4.0

https://www.home-assistant.io/blog/2025/04/02/release-20254

dotlambda committed:
python313Packages.aioesphomeapi: 29.4.0 -> 29.8.0

Diff: https://github.com/esphome/aioesphomeapi/compare/refs/tags/v29.4.0...v29.8.0

Changelog: https://github.com/esphome/aioesphomeapi/releases/tag/v29.8.0

dotlambda committed:
python313Packages.bluetooth-auto-recovery: 1.4.4 -> 1.4.5

Diff: https://github.com/Bluetooth-Devices/bluetooth-auto-recovery/compare/refs/tags/v1.4.4...v1.4.5

Changelog: https://github.com/Bluetooth-Devices/bluetooth-auto-recovery/blob/v1.4.5/CHANGELOG.md

dotlambda committed:
python313Packages.ha-silabs-firmware-client: init at 0.2.0

dotlambda committed:
home-assistant.python.pkgs.home-assistant-frontend: 20250306.0 -> 20250401.0

Changelog: https://github.com/home-assistant/frontend/releases/tag/20250312.0
           https://github.com/home-assistant/frontend/releases/tag/20250326.0
           https://github.com/home-assistant/frontend/releases/tag/20250327.0
           https://github.com/home-assistant/frontend/releases/tag/20250327.1
           https://github.com/home-assistant/frontend/releases/tag/20250328.0
           https://github.com/home-assistant/frontend/releases/tag/20250331.0
           https://github.com/home-assistant/frontend/releases/tag/20250401.0

dotlambda committed:
home-assistant.python.pkgs.home-assistant-intents: 2025.3.5 -> 2025.3.28

Diff: https://github.com/home-assistant/intents-package/compare/refs/tags/2025.3.5...2025.3.28

Changelog: https://github.com/home-assistant/intents/releases/tag/2025.3.12
           https://github.com/home-assistant/intents/releases/tag/2025.3.23
           https://github.com/home-assistant/intents/releases/tag/2025.3.24
           https://github.com/home-assistant/intents/releases/tag/2025.3.28

dotlambda committed:
python313Packages.hdate: 0.11.1 -> 1.0.3

Diff: https://github.com/py-libhdate/py-libhdate/compare/refs/tags/v0.11.1...v1.0.3

Changelog: https://github.com/py-libhdate/py-libhdate/blob/v1.0.3/CHANGELOG.md

trofi committed:
love_0_10: add missing xorg.libX11 depend

SDL2 intentionally dropped xorg.libX11 propagated build depend.
This broke `love_0_10` as:

    In file included from modules/window/sdl/Window.cpp:43:
    /nix/store/nhq38i6s7zyhaa4l9csp5d1f1hyfzdqa-sdl2-compat-2.32.52-dev/include/SDL2/SDL_syswm.h:68:10: fatal error: X11/Xlib.h: No such file or directory
       68 | #include <X11/Xlib.h>

Let's add the dependency to `love_0_10` instead.

dotlambda committed:
python313Packages.dbus-fast: 2.33.0 -> 2.44.1

Diff: https://github.com/Bluetooth-Devices/dbus-fast/compare/refs/tags/v2.33.0...v2.44.1

Changelog: https://github.com/Bluetooth-Devices/dbus-fast/blob/v2.44.1/CHANGELOG.md

fabaff committed:
python313Packages.fyta-cli: 0.7.0 -> 0.7.2

Diff: https://github.com/dontinelli/fyta_cli/compare/refs/tags/v0.7.0...v0.7.2

Changelog: https://github.com/dontinelli/fyta_cli/releases/tag/v0.7.1
           https://github.com/dontinelli/fyta_cli/releases/tag/v0.7.2

dotlambda committed:
python313Packages.habluetooth: 3.25.0 -> 3.38.0

Diff: https://github.com/Bluetooth-Devices/habluetooth/compare/refs/tags/v3.25.0...v3.38.0

Changelog: https://github.com/Bluetooth-Devices/habluetooth/blob/v3.38.0/CHANGELOG.md

dotlambda committed:
python313Packages.hass-nabucasa: 0.92.0 -> 0.95.0

Diff: https://github.com/nabucasa/hass-nabucasa/compare/refs/tags/0.92.0...0.95.0

Changelog: https://github.com/NabuCasa/hass-nabucasa/releases/tag/0.95.0

dotlambda committed:
python313Packages.mastodon-py: 1.8.1 -> 2.0.1

Diff: https://github.com/halcy/Mastodon.py/compare/1.8.1...v2.0.1

Changelog: https://github.com/halcy/Mastodon.py/blob/v2.0.1/CHANGELOG.rst

dotlambda committed:
python313Packages.music-assistant-models: 1.1.30 -> 1.1.43

Diff: https://github.com/music-assistant/models/compare/refs/tags/1.1.30...1.1.43

Changelog: https://github.com/music-assistant/models/blob/1.1.43/CHANGELOG.md

dotlambda committed:
python313Packages.music-assistant-client: 1.1.1 -> 1.2.0

Diff: https://github.com/music-assistant/client/compare/refs/tags/1.1.1...1.2.0

Changelog: https://github.com/music-assistant/client/blob/1.2.0/CHANGELOG.md

dotlambda committed:
python313Packages.python-linkplay: 0.2.1 -> 0.2.2

Diff: https://github.com/Velleman/python-linkplay/compare/refs/tags/v0.2.1...v0.2.2

Changelog: https://github.com/Velleman/python-linkplay/releases/tag/v0.2.2

dotlambda committed:
python313Packages.python-tado: 0.18.6 -> 0.18.9

Diff: https://github.com/wmalgadey/PyTado/compare/refs/tags/0.18.6...0.18.9

Changelog: https://github.com/wmalgadey/PyTado/releases/tag/0.18.9

dotlambda committed:
python313Packages.tesla-fleet-api: 0.9.13 -> 1.0.17

Diff: https://github.com/Teslemetry/python-tesla-fleet-api/compare/refs/tags/v0.9.13...v1.0.17

dotlambda committed:
python313Packages.zha: 0.0.53 -> 0.0.55

Diff: https://github.com/zigpy/zha/compare/refs/tags/0.0.53...0.0.55

Changelog: https://github.com/zigpy/zha/releases/tag/0.0.54
           https://github.com/zigpy/zha/releases/tag/0.0.55

dotlambda committed:
python313Packages.zigpy: 0.77.1 -> 0.78.0

Diff: https://github.com/zigpy/zigpy/compare/refs/tags/0.77.1...0.78.0

Changelog: https://github.com/zigpy/zigpy/releases/tag/0.78.0

dotlambda committed:
python313Packages.zigpy-znp: 0.13.1 -> 0.14.0

Diff: https://github.com/zigpy/zigpy-znp/compare/refs/tags/v0.13.1...v0.14.0

Changelog: https://github.com/zigpy/zigpy-znp/releases/tag/v0.14.0

dotlambda committed:
python313Packages.bellows: 0.43.0 -> 0.44.0

Diff: https://github.com/zigpy/bellows/compare/refs/tags/0.43.0...0.44.0

Changelog: https://github.com/zigpy/bellows/releases/tag/0.44.0

dotlambda committed:
python313Packages.zha-quirks: 0.0.134 -> 0.0.136

Diff: https://github.com/zigpy/zha-device-handlers/compare/refs/tags/0.0.134...0.0.136

Changelog: https://github.com/zigpy/zha-device-handlers/releases/tag/0.0.135
           https://github.com/zigpy/zha-device-handlers/releases/tag/0.0.136

dotlambda committed:
python313Packages.zwave-js-server-python: 0.61.0 -> 0.62.0

Diff: https://github.com/home-assistant-libs/zwave-js-server-python/compare/refs/tags/0.61.0...0.62.0

Changelog: https://github.com/home-assistant-libs/zwave-js-server-python/releases/tag/0.62.0

raboof committed:
jemalloc: fix cross-compilation

Follow-up on https://github.com/NixOS/nixpkgs/pull/393724

Not *sure* why this works, but following the lead of `jbig2dec`

SuperSandro2000 committed:
python313Packages.python-cinderclient: fix src pname

SuperSandro2000 committed:
python313Packages.python-keystoneclient: fix src pname

SuperSandro2000 commented:
#396017

mweinelt committed:
Fix openstack (#396017)

SuperSandro2000 committed:
python313Packages.oslo-concurrency: fix src pname

SuperSandro2000 committed:
python313Packages.oslo-db: fix src pname

SuperSandro2000 committed:
python313Packages.python-glanceclient: fix src pname

SuperSandro2000 committed:
python313Packages.python-manilaclient: fix src pname

SuperSandro2000 committed:
python313Packages.python-novaclient: fix src pname

SuperSandro2000 committed:
python313Packages.python-octaviaclient: fix src pname

SuperSandro2000 committed:
python313Packages.python-swiftclient: fix src pname

SuperSandro2000 committed:
python313Packages.osc-lib: 3.1.0 -> 3.2.0

dotlambda committed:
python313Packages.passlib: turn into wrapper around libpass

dotlambda committed:
Revert "python313Packages.bundlewrap: drop passlib constraint"

This reverts commit a97b72577880b9658ed83dc5da5595582a9ed7a7.

dotlambda committed:
python313Packages.mitmproxy: relax h2 version constraint

dotlambda committed:
Revert "mailman: remove passlib constraint"

This reverts commit 0c1b9474d86bbca1fad0ce7288115b6547aaaf9e.

dotlambda committed:
python313Packages.libpass: init at 1.9.0

drupol committed:
jemalloc: cross compile (#395996)

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
rustc: fix fastCross

1.86 changed the naming scheme of the stamp files.

7c6f434c committed:
love_0_10: add missing xorg.libX11 depend (#396117)

mweinelt committed:
python313Packages.pushover-complete: fix and modernize build

mweinelt committed:
python313Packages.passlib: restore old name in distinfo (#396065)

nixpkgs-ci[bot] committed:
Merge master into staging-next

mweinelt committed:
python313Packages.slixmpp: fix build and modernize

mweinelt committed:
home-assistant.python.pkgs.pytest-homeassistant-custom-component: 0.13.223 -> 0.13.232

https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/blob/refs/tags/0.13.232/CHANGELOG.md

mweinelt committed:
nixos/tests/home-assistant: update for 2025.4.0

Remove the shelly component, as it now has a dependency on a bluetooth
setup, which is not easy to provide in the test environment. Enabling
`hardware.bluetooth.enable` is insufficient.

Turn up the loglevel to `debug`, so we can still see which integrations
get set up. This is used for for checking whether all our ways of setting
up integrations still work.

mweinelt committed:
home-assistant-custom-components.emporia_vue: 0.10.1 -> 0.10.2-pre

https://github.com/magico13/ha-emporia-vue/releases/tag/v0.10.2-pre

mweinelt committed:
home-assistant-custom-components.ntfy: 1.2.0-pre.1 -> 1.2.0-pre.2

https://github.com/hbrennhaeuser/homeassistant_integration_ntfy/compare/v1.2.0-pre.1...v1.2.0-pre.2

mweinelt committed:
home-assistant-custom-components.solax_modbus: 2025.02.1 -> 2025.04.1

https://github.com/wills106/homeassistant-solax-modbus/releases/tag/2025.04.1

mweinelt committed:
home-assistant-custom-components.solis-sensor: 3.9.0 -> 3.9.1

https://github.com/hultenvp/solis-sensor/releases/tag/v3.9.1

mweinelt committed:
home-assistant-custom-components.tuya_local: 2025.2.2 -> 2025.3.0

https://github.com/make-all/tuya-local/releases/tag/2025.3.0

mweinelt committed:
home-assistant-custom-components.yoto_ha: 1.23.2 -> 1.24.0

https://github.com/cdnninja/yoto_ha/releases/tag/v1.24.0

mweinelt committed:
home-assistant-custom-lovelace-modules.clock-weather-card: 2.8.10 -> 2.8.11

https://github.com/pkissling/clock-weather-card/blob/v2.8.11/CHANGELOG.md

mweinelt committed:
home-assistant-custom-lovelace-modules.mini-graph-card: 0.12.1 -> 0.13.0-dev.2

https://github.com/kalkih/mini-graph-card/releases/tag/v0.13.0-dev.2

mweinelt committed:
home-assistant: 2025.3.4 -> 2025.4.0 (#395618)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
valkey: skip more flakey tests

K900 committed:
qt6.qtdeclarative: add sigtool on Darwin

Needed since 6.9

LordGrimmauld committed:
python312Packages.dist-meta: fix build

LordGrimmauld committed:
python312Packages.shippinglabel: fix build

LordGrimmauld committed:
python312Packages.pyproject-parser: fix build

K900 committed:
python312Packages.shippinglabel: fix build (#396272)

K900 committed:
python312Packages.dist-meta: fix build (#396269)

K900 committed:
python312Packages.pyproject-parser: fix build (#396274)

LordGrimmauld committed:
python312Packages.dist-meta: fix hatch to be in build-system

LordGrimmauld committed:
python312Packages.pyproject-parser: fix hatch to be in build-system

LordGrimmauld committed:
python312Packages.shippinglabel: fix hatch to be in build-system

K900 committed:
python312Packages.{dist-meta, pyproject-parser, shippinglabel}: move hatch to build-system (#396303)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
ffmpeg_4: backport build fix

K900 committed:
python312Packages.celery: add some optdepends

These are required for tests, and are currently pulled in
mostly via accidental propagation.

K900 committed:
python312Packages.kazoo: remove old unused test stuff

K900 committed:
zellij: fix build with Rust 1.86

K900 committed:
qbittorrent: fix build with Qt 6.9

K900 committed:
kdePackages.kdeconnect-kde: fix build with Qt 6.9

K900 committed:
telegram-desktop: fix build with Qt 6.9

K900 committed:
openldap: skip flaky test

For some reason, it's exceptionally flaky on i686 specifically,
but it is _also_ flaky on more normal targets.

K900 committed:
python312Packages.afdko: backport patch to fix some tests, skip tests it doesn't fix

I really don't have it in me to bisect this thing right now.

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
python312Packages.afdko: reformat

K900 committed:
nodejs_18: hack to fix build

It should be dropped soon.

mweinelt committed:
python313Packages.python-tado: 0.18.9 -> 0.18.11

https://github.com/wmalgadey/PyTado/releases/tag/0.18.11

mweinelt committed:
python313Packages.bring-api: 1.0.2 -> 1.1.0

https://github.com/miaucl/bring-api/releases/tag/1.1.0

mweinelt committed:
home-assistant: 2025.4.0 -> 2025.4.1

https://github.com/home-assistant/core/releases/tag/2025.4.1

mweinelt committed:
home-assistant.frontend: 20250401.0 -> 2025.0404.0

https://github.com/home-assistant/frontend/releases/tag/20250404.0

mweinelt committed:
home-assistant: 2025.4.0 -> 2025.4.1 (#396441)

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
python312Packages.pyqt6: 6.8.0 -> 6.9.0 (?)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

mweinelt committed:
python3Packages.soundcard: fix src pname

LordGrimmauld committed:
python312Packages.pygame: pin to SDL2_classic

Some niche regressions in sdl2-compat over SDL2 have pygame tests fail.
Some of these were adressed in https://github.com/libsdl-org/sdl2-compat/issues/367,
but pygame still does not yet work with sdl2-compat.
Pinning is the simplext solution for now.

K900 committed:
python312Packages.pygame: pin to SDL2_classic (#396561)

vcunat committed:
gst_all_1,mopidy: Fix overridden builds (#395981)

mweinelt committed:
python3Packages.instructor: relax rich constraint

TomaSajt committed:
electron_{33,34}: fix build by adding patch to use adler2 lib

LordGrimmauld committed:
inochi-*: add missing libGL and libX11

LordGrimmauld committed:
cro-mag-rally: add missing libGL

LordGrimmauld committed:
endless-sky: add missing libX11

LordGrimmauld committed:
jaz2: add missing libGL

LordGrimmauld committed:
nanosaur2: add missing libGL

LordGrimmauld committed:
neverball: add missing libX11

LordGrimmauld committed:
opensoldat: add missing libX11

LordGrimmauld committed:
pioneer: add missing libX11

LordGrimmauld committed:
q2pro: add missing libX11 and libGL

LordGrimmauld committed:
syncterm: add missing libX11

LordGrimmauld committed:
vdrift: add missing libX11

LordGrimmauld committed:
vivictpp: add missing libX11

LordGrimmauld committed:
vpv: add missing libGL and libX11

LordGrimmauld committed:
bugdom: add missing libGL

LordGrimmauld committed:
cataclysm-dda: add missing libX11

LordGrimmauld committed:
quakespasm: add missing libX11

LordGrimmauld committed:
warsow: add missing libX11

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
python312Packages.pyside6: 6.8.0.2 -> 6.9.0

K900 committed:
treewide: add back missing libGL and libX11 for packages depending on SDL2 (#396612)

veprbl committed:
wafHook: run waf without explicit python interpreter

booxter committed:
awscli2: relax ruamel.yaml.clib dependency

It was bumped in c861a11e7174492dbf29db75348d0d3777d6fb62 beyond the
upstream cap. It should be ok to remove the cap since the dependency is
applicable for python 3.10 and earlier, as per the comment in the
patched file:

https://github.com/aws/aws-cli/blob/c24bf195ca8cdab9ed59df0d1f8b47185afaffa0/pyproject.toml#L37

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

booxter committed:
gst_all_1.gst-plugins-base: fix darwin build

Disable `xi` and `xvideo` since the platform is not using X11.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

booxter committed:
libcbor: fix cmake build on case-insensitive FS (on darwin)

The tarball already contains a `BUILD` file (not directory), which makes
the cmake configure implementation to fail when creating the `build`
directory. This is happening because for some file systems (e.g. on
darwin), file names that differ only in case are considered duplicate.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

veprbl committed:
wafHook: do patchShebangs on $wafPath

booxter committed:
python312Packages.pycardano: fix build

It pinned `ogmios` that was bumped recently.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

vcunat committed:
wafHook: run waf without explicit python interpreter (#396641)

nixpkgs-ci[bot] committed:
Merge master into staging-next

vcunat committed:
libcbor: avoid rebuilding on linux for now

CI says
> Rebuild: linux 15669, darwin 415

emilylange committed:
[staging-next] electron_{33,34}: fix build by adding patch to use adler2 lib (#396606)

transcaffeine committed:
victoriametrics: 1.113.0 -> 1.115.0

Release notes: https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.115.0
Full changelog: https://github.com/VictoriaMetrics/VictoriaMetrics/compare/v1.113.0...v1.115.0

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

NickCao committed:
python312Packages.pycardano: fix build (#396715)

booxter committed:
nodejs*: fix test failure on darwin on latest MacOS 15.4

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
treewide: fix mismerge

K900 committed:
python312Packages.alarmdecoder: fix more mismerge

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
lxqt.libqtxdg: fix build with Qt 6.9

Why. WHY.

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
nodejs*: fix test failure on darwin on latest MacOS 15.4 (#396958)

K900 committed:
gst_all_1.gst-plugins-base: fix darwin build (#396658)

K900 committed:
libcbor: fix cmake build on case-insensitive FS (on darwin) (#396662)

K900 committed:
zulu: fix man path on Darwin

aduh95 committed:
nodejs: skip tests that are failing on macOS 15.4

nixpkgs-ci[bot] committed:
Merge master into staging-next

leona-ya committed:
victoriametrics: 1.113.0 -> 1.115.0 (#396856)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

r-vdp committed:
atuin: add update script

nixpkgs-ci[bot] committed:
Merge master into staging-next

r-vdp committed:
atuin: 18.4.0 -> 18.5.0

booxter committed:
python312Packages.thinc: fix build; modernize; remove old darwin crust

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

SuperSandro2000 committed:
atuin: 18.4.0 -> 18.5.0 (#397476)

dotlambda committed:
python313Packages.pyheos: 1.0.4 -> 1.0.5

Diff: https://github.com/andrewsayre/pyheos/compare/refs/tags/1.0.4...1.0.5

Changelog: https://github.com/andrewsayre/pyheos/releases/tag/1.0.5

nixpkgs-ci[bot] committed:
Merge master into staging-next

vcunat committed:
awscli2: relax ruamel.yaml.clib dependency (#396653)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
python312Packages.thinc: fix build; modernize; remove old darwin crust (#397480)

K900 committed:
qt5: switch to Apple SDK 14 on -darwin platforms

Fixes weird linking errors seen with SDK 13.

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
opencv4: remove old Darwin cruft, use SDK 14

Same issue as Qt5.

K900 committed:
opencv4: reformat

K900 committed:
libepoxy: remove old Darwin gunk, re-skip test that is broken, use replace-fail

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

booxter committed:
python312Packages.bokeh: fix build (add missing runtime dependency)

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

K900 committed:
python312Packages.bokeh: fix build (add missing runtime dependency) (#397737)

booxter committed:
python312Packages.cramjam: disable broken test

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

K900 committed:
python312Packages.cramjam: disable broken test (#397754)

K900 committed:
kdePackages.kpimtextedit: fix build with Qt 6.9

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
kdePackages.plasma-vault: fix build with Qt 6.9

K900 committed:
kdePackages.marble: fix build with Qt 6.9

K900 committed:
kdePackages.falkon: fix build with PySide 6.9

dotlambda committed:
python313Packages.pyheos: fix tests on Darwin

mweinelt committed:
python313Packages.django-extensions: 3.2.3 -> 4.0

https://github.com/django-extensions/django-extensions/releases/tag/4.0

nixpkgs-ci[bot] committed:
Merge master into staging-next

mweinelt committed:
pretix: relax celery constraint

mweinelt committed:
python313Packages.django-extensions: 3.2.3 -> 4.0 (#397831)

nixpkgs-ci[bot] committed:
Merge master into staging-next

alyssais committed:
lib.systems: force qemuArch to "aarch64" on aarch64

qemu architecture names are fixed — we're using uname here just
because it's more likely to be correct than CPU name (see e.g. POWER).
This means that aarch64 is always called aarch64, even on Darwin where
uname reports arm64.

Fixes: 61582c704327 ("lib/systems: use Darwin architecture names for `config` and `uname`")

emilazy committed:
[staging-next] lib.systems: force qemuArch to "aarch64" on aarch64 (#397889)

K900 committed:
wxGTK32: 3.2.6 -> 3.2.7.1

K900 committed:
python312Packages.wxpython: 4.2.2 -> 4.2.3, fix build

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

booxter committed:
gst_all_1.gst-plugins-bad: fix aarch64-darwin build

The package requires a newer SDK to access
`kVTCompressionPropertyKey_ConstantBitRate`.

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

emilazy committed:
gst_all_1.gst-plugins-bad: fix aarch64-darwin build (#398009)

NickCao committed:
python313Packages.pyheos: 1.0.4 -> 1.0.5 (#397525)

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

mweinelt committed:
python313Packages.aioairzone: 0.9.9 -> 1.0.0

https://github.com/Noltari/aioairzone/releases/tag/1.0.0

mweinelt committed:
python313Packages.aioesphomeapi: 29.8.0 -> 29.9.0

https://github.com/esphome/aioesphomeapi/releases/tag/v29.9.0

mweinelt committed:
python313Packages.aioshelly: 13.4.0 -> 13.4.1

https://github.com/home-assistant-libs/aioshelly/releases/tag/13.4.1

mweinelt committed:
python313Packages.ical: 9.0.3 -> 9.1.0

https://github.com/allenporter/ical/releases/tag/9.1.0

mweinelt committed:
python313Packages.livisi: 0.0.24 -> 0.0.25

https://github.com/planbnet/livisi/compare/refs/tags/v0.0.24...refs/tags/v0.0.25

mweinelt committed:
python313Packages.opower: 0.10.1 -> 0.11.1

https://github.com/tronikos/opower/releases/tag/v0.11.1

mweinelt committed:
python313Packages.pyvicare: 2.43.1 -> 2.44.0

https://github.com/openviess/PyViCare/releases/tag/2.44.0

mweinelt committed:
python313Packages.reolink-aio: 0.13.0 -> 0.13.1

https://github.com/starkillerOG/reolink_aio/releases/tag/0.13.1

mweinelt committed:
python313Packages.sharkiq: 1.0.2 -> 1.1.0

https://github.com/JeffResc/sharkiq/compare/refs/tags/v1.0.2...refs/tags/v1.1.0

vcunat commented:
- [ ] deepin trouble
  https://hydra.nixos.org/build/294358823
  maintainer: @wineee

vcunat committed:
Revert "python3Packages.jsonrpclib-pelix: 0.4.3.3 -> 0.4.3.4"

This reverts commit a1f43b177e86a2f5adfb313c4fcd36431b662046.
It wouldn't build:
https://hydra.nixos.org/build/294234272/nixlog/3/tail

mweinelt committed:
home-assistant: 2025.4.1 -> 2025.4.2

https://github.com/home-assistant/core/releases/tag/2025.4.2

vcunat commented:
- [ ] `python3Packages.wandb` regressed some tests
  https://hydra.nixos.org/build/294645255/nixlog/2/tail
  maintainer: @samuela

vcunat committed:
python3Packages.strawberry-graphql: 0.258.0 -> 0.263.1

https://hydra.nixos.org/build/294599100/nixlog/3/tail

vcunat committed:
python3Packages.playwright: don't apply libavif patches

Taking a custom libavif commit and applying patches
meant for another version - that was just bound to break.
https://hydra.nixos.org/build/294635352/nixlog/2/tail

vcunat committed:
Revert "python3Packages.osqp: 0.6.7.post3 -> 1.0.1"

This reverts commit e8ff8ef3764d4d87eca72ddbc2105efbbed925be.
The revert fixes the build.
Unfortunately it's not as easy as just dropping the postPatch.

mweinelt committed:
python3Packages.wandb: skip sentry tests

Tests regressed after an update of the sentry-sdk.

https://hydra.nixos.org/build/294645255/nixlog/2/tail

mweinelt committed:
python313Packages.homeassistant-stubs: 2025.3.4 -> 2025.4.2

https://github.com/KapJI/homeassistant-stubs/releases/tag/2025.4.2

mweinelt committed:
home-assistant.python.pkgs.pytest-homeassistant-custom-component: 0.13.232 -> 0.13.233

https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/blob/refs/tags/0.13.233/CHANGELOG.md

mweinelt committed:
python313Packages.gcal-sync: backport ical v9.0 compat

nixpkgs-ci[bot] committed:
Merge master into staging-next

Yarny0 commented:
@vcunat : Would https://github.com/NixOS/nixpkgs/pull/397941 be an option to restore the `python3Packages.jsonrpclib-pelix` update to 0.4.3.4 ?

vcunat commented:
Sure, whatever works.  I only noticed the PR after I pushed the revert.

vcunat committed:
python3Packages.playwright: nixfmt

mweinelt committed:
python3Packages.piano-transcription-inference: fix and modernize build

mweinelt committed:
python3Packages.azure-mgmt-kusto: fix src pname

mweinelt committed:
python3Packages.flask-pymongo: fix src pname

and migrate to PEP517 builder.

mweinelt committed:
python3Package.pytest-testinfra: fix src pname

and update the build-system.

mweinelt committed:
python3Packages.zope-security: add missing test deps

mweinelt committed:
python3Packages.zope-size: fix build

Fetch from GitHub, add missing test deps, relasx setuptools constraint.

mweinelt committed:
python3Packages.commoncode: relax beautifulsoup4 constraint

mweinelt committed:
python3Packages.s3fs: relax fsspec constraint

mweinelt committed:
python3Packages.dbt-adapters: drop obsolete postPatch phase

mweinelt committed:
python3Packages.psygnal: ignore pydantic deprecation warnings

mweinelt committed:
python3Packages.zodbpickle: fix build

mweinelt committed:
python3Packages.willow: disable failing test

This is related to the AVIF migration from pillow-heif to mainline
pillow.

mweinelt committed:
home-assistant: 2025.4.1 -> 2025.4.2 (#398160)

nixpkgs-ci[bot] committed:
Merge master into staging-next

K900 committed:
Merge remote-tracking branch 'origin/master' into staging-next

nixpkgs-ci[bot] committed:
Merge master into staging-next

vcunat committed:
release.nix: don't block on aarch64-darwin bootstrap for now

K900 committed:
musescore: fix build with Qt 6.9

K900 committed:
kdePackages.libktorrent: fix build with Qt 6.9

nixpkgs-ci[bot] committed:
Merge master into staging-next

vcunat committed:
nodejs: skip tests that are failing on macOS 15.4 (#397172)

vcunat commented:
- [ ] `nixosTests.gitlab` regressed in this PR, blocking the `nixos-unstable` channel now
  ```
  gitlab: must succeed: mkdir -m 700 /root/.ssh
  gitlab # mkdir: cannot create directory ‘/root/.ssh’: File exists
  ```
  maintainers: @globin, @yayayayaka
  Hydra e.g. https://hydra.nixos.org/build/294952619
